### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by LukasHromadnik**
Tests for correct translation of properties to attributes

### CODE REVIEW

File: BetterLabelTests/BetterLabelTests.swift [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+//  BetterLabelTests.swift
+//  BetterLabelTests
+//
+//  Created by Jakub Olejník on 25/10/2018.
+//  Copyright © 2018 Jakub Olejník. All rights reserved.
+//
+
+import XCTest
+
+class BetterLabelTests: XCTestCase {
```
Comment (olejnjak) [inline BetterLabelTests/BetterLabelTests.swift]: What can be `final` should be `final`

File: BetterLabelTests/BetterLabelTests.swift [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
```
Comment (olejnjak) [inline BetterLabelTests/BetterLabelTests.swift]: Please remove this empty line so it matches the style of the whole project.

File: BetterLabelTests/BetterLabelTests.swift [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        let attributedText = label.attributedText
+        XCTAssertEqual(attributedText.string, "Lorem ipsum")
+    }
+    
+    // MARK: - Helpers
+    
+    private func loadAttribute(_ attribute: NSAttributedString.Key) -> Any? {
+        return label.attributedText.attribute(attribute, at: 0, effectiveRange: nil)
+    }
+
```
Comment (olejnjak) [inline BetterLabelTests/BetterLabelTests.swift]: Please remove this empty line so it matches the style of the whole project.

**Review by olejnjak — CHANGES_REQUESTED** [review]

**Comment by LukasHromadnik** [comment]
It should be all fixed

**Pull request merged**